frontend/vite-prod.config.ts

```diff
@@ -40,17 +43,17 @@ export default defineConfig(({ command }) => {
       emptyOutDir: true,
       sourcemap: true,
       minify: false,
+      // cssMinify: 'lightningcss',
       rollupOptions: {
         input: {
           app: "index.html",
         },
       },
     },
     css: {
-      modules: {
-        // Only dashes in class names will be converted to camelCase,
-        // the original class name will not to be removed from the locals
-        localsConvention: "dashes",
+      transformer: "lightningcss",
+      lightningcss: {
+        targets: browserslistToTargets(browserslist(">= 0.25%")),
```

Comment:
[This includes ancient browsers such as IE 11](https://browserslist.dev/?q=PiAwLjI1JQ%3D%3D)

Reply:
Fixed in a91d35396e67b20d8a641f240b9de9742187151b